package.json

```diff
@@ -1,7 +1,11 @@
 {
   "name": "elasticsearch-node-rotation",
   "dependencies": {
-    "aws-sdk": "^2.1142.0"
+    "@aws-sdk/client-auto-scaling": "^3.405.0",
+    "@aws-sdk/client-ec2": "^3.406.0",
```

Comment:
Any reason this is a different version from other `@aws-sdk` libraries?

Reply:
ooh, good spot. I think that must be an accident relating to the fact i installed them with `npm` and didn't do them all at the same time

Reply:
I've bumped them all to 3.408.0 as that's the latest version